keyboards/lily58/keymaps/default/config.h

```diff
@@ -28,19 +28,13 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 // #define MASTER_RIGHT
 // #define EE_HANDS
 
+#define SSD1306OLED
```

Comment:
Honestly, I'd really prefer to use the `oled_driver` feature, rather than this.

Reply:
thanks. I accept the change.